local_examples/cmds_hash/predis/CmdsHashTest.php

```diff
@@ -0,0 +1,133 @@
+<?php
```

Comment:
It looks odd, since it would appear to print "// EXAMPLE: cmds_hash" (ie, it's not a comment when it's outside the `<?php` section), but we still need it to activate the example. The tab isn't currently visible in staging, but it should work when this is added.
```suggestion
// EXAMPLE: cmds_hash
<?php
```